README.md

```diff
@@ -79,3 +79,23 @@ Here's some suggestions of QOL things you should do for minigames, they are **no
 * It's recommended to keep player colors consistent between games. We set some definitions in `core.h` which you should use. 
 * We have button icons available in `assets/core` (we're missing some, working on it)
 * Try to keep your (compressed) assets under 2 MiB, since everyone needs to share the ROM space. Not a big deal if you MUST go over.
+
+### Building with libdragon-cli
+
+Clone this repository with `--recurse-submodules` or if you haven't run:
+
+```bash
+git submodule update --init
+```
+
+Init container:
+
+```bash
+libdragon init
+```
+
+Build:
+
+```bash
+libdragon make T3D_INST=./tiny3d
```

Comment:
Once we have a proper "plugin" system, this will become unnecessary in the future